scripts/core/context.yml

```diff
@@ -98,6 +98,14 @@ etors:
       desc: "[bool] to indicate if the $xEnqueueUSMMemcpy2D entrypoint is supported."
     - name: USM_FILL2D_SUPPORT
       desc: "[bool] to indicate if the $xEnqueueUSMFill2D entrypoint is supported."
+    - name: ATOMIC_MEMORY_ORDER_CAPABILITIES
+      desc: "[$x_memory_order_capability_flags_t] return a bit-field of atomic memory order capabilities."
+    - name: ATOMIC_MEMORY_SCOPE_CAPABILITIES
+      desc: "[x_memory_scope_capability_flags_t] return a bit-field of atomic memory scope capabilities."
+    - name: ATOMIC_FENCE_ORDER_CAPABILITIES
+      desc: "[$x_memory_order_capability_flags_t] return a bit-field of atomic memory fence order capabilities."
+    - name: ATOMIC_FENCE_SCOPE_CAPABILITIES
+      desc: "[x_memory_scope_capability_flags_t] return a bit-field of atomic memory fence scope capabilities."
```

Comment:
I'd expect `0` to be returned if context level atomic fences aren't supported, this is only implemented in the CUDA adapter currently I believe. Perhaps we should mention this in the descriptions?

Reply:
👍